src/Form/MediaType.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace App\Form;
+
+use App\Entity\TrickMedia;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\FileType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\Extension\Core\Type\UrlType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormError;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\File;
+use Symfony\Component\Validator\Constraints\Length;
+use Symfony\Component\Validator\Constraints\NotBlank;
+use Symfony\Component\Validator\Constraints\Url;
+
+class MediaType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('alt', TextType::class, [
+                'attr' => ['class' => 'form-control'],
+                'label' => 'Description de l\'image (Alt)',
+                'constraints' => [
+                    new Length([
+                        'normalizer' => 'trim',
+                        'min' => 3,
+                        'max' => 200,
+                        'minMessage' => 'La description (Alt) est trop courte, minimum {{ limit }} caractères',
+                        'maxMessage' => 'La description (Alt) est trop longue, maximum {{ limit }} caractères',
+                    ]),
+                    new NotBlank(['message' => TrickType::NOTEMPTY_MESSAGE])
+                ],
+            ])
+            ->addEventListener(FormEvents::POST_SET_DATA, function (FormEvent $event) {
+                /** @var TrickMedia $media */
+                $media = $event->getData();
+                $form = $event->getForm();
+
+                // When we create a new media
+                if (!$media) {
+                    $form->add('image', FileType::class, [
```

Comment:
What if média exists and is a file ?

Reply:
!$media is set to false if a media exist. 
When a media exist, and it is an image, the fileType field isn't added, a user can only edit the alt attribute of a existing image.
But since !$media is set to true when no media exist, the field will be added when creating a new media (from edit trick page or when creating a new trick)